tests/python_client/testcases/test_insert.py

```diff
@@ -1201,3 +1201,36 @@ def test_insert_string_field_auto_id_is_true(self):
         df = [int_field, string_field, vec_field]
         error = {ct.err_code: 0, ct.err_msg: 'Data type is not support.'}
         collection_w.insert(data=df, check_task=CheckTasks.err_res, check_items=error)
+
+    @pytest.mark.tags(CaseLabel.L1)
+    @pytest.mark.skip(reason="Insert empty string will panic, need to wait for fix")
```

Comment:
do we have an issue? Could you add the issue here?

Reply:
There is no issue for master, the issue is for 2.1 image，do we need to add it?

Reply:
Yes, if the problem is the same.